sources/web/datalab/server.ts

```diff
@@ -244,7 +244,9 @@ function uncheckedRequestHandler(request: http.ServerRequest, response: http.Ser
       urlpath.indexOf('/oauthcallback') == 0) {
     // Start or return from auth flow.
     auth.handleAuthFlow(request, response, parsed_url, appSettings);
-  } else if ((urlpath.indexOf('/static') == 0) || (urlpath.indexOf('/custom') == 0)) {
+  } else if (urlpath.indexOf('/static') == 0 ||
```

Comment:
Maybe you should have a method static_.isStaticResource(urlpath) that does this three-way check.